ios/FluentUI/People Picker/ContactView.swift

```diff
@@ -0,0 +1,172 @@
+            }
+
+            constraints.append(lastNameLabel.leadingAnchor.constraint(equalTo: self.leadingAnchor))
+            constraints.append(lastNameLabel.trailingAnchor.constraint(equalTo: self.trailingAnchor))
+            constraints.append(lastNameLabel.widthAnchor.constraint(equalTo: self.widthAnchor))
+            constraints.append(lastNameLabel.bottomAnchor.constraint(equalTo: self.bottomAnchor))
+        }
+        return constraints
+    }
+
+    private func identifierLayoutConstraints() -> [NSLayoutConstraint] {
+        var constraints = [NSLayoutConstraint]()
+        if let identifierLabel = identifierLabel {
+            addSubview(identifierLabel)
+            identifierLabel.translatesAutoresizingMaskIntoConstraints = false
+            constraints.append(identifierLabel.widthAnchor.constraint(equalTo: self.widthAnchor))
+            constraints.append(identifierLabel.centerXAnchor.constraint(equalTo: self.centerXAnchor))
+            constraints.append(identifierLabel.topAnchor.constraint(equalTo: avatarView.bottomAnchor, constant: Constants.spacingBetweenAvatarAndFirstLabel))
+            constraints.append(identifierLabel.leadingAnchor.constraint(equalTo: self.leadingAnchor))
+            constraints.append(identifierLabel.trailingAnchor.constraint(equalTo: self.trailingAnchor))
+            constraints.append(identifierLabel.bottomAnchor.constraint(equalTo: self.bottomAnchor))
+        }
+        return constraints
+    }
+
+    private func setupFirstNameLabel(using firstName: String) {
+        firstNameLabel = UILabel(frame: .zero)
+
+        if let firstNameLabel = firstNameLabel {
+            firstNameLabel.adjustsFontForContentSizeCategory = true
+            firstNameLabel.font = Fonts.subhead
+            firstNameLabel.text = firstName
+            firstNameLabel.textColor = Colors.Contact.firstName
+            firstNameLabel.textAlignment = .center
+        }
+    }
+
+    private func setupLastNameLabel(using lastName: String) {
+        lastNameLabel = UILabel(frame: .zero)
+
+        if let lastNameLabel = lastNameLabel {
+            lastNameLabel.adjustsFontForContentSizeCategory = true
+            lastNameLabel.font = Fonts.footnote
+            lastNameLabel.text = lastName
+            lastNameLabel.textAlignment = .center
+            lastNameLabel.textColor = Colors.Contact.lastName
+        }
+    }
+
+    private func setupIdentifierLabel(using identifier: String) {
+        identifierLabel = UILabel(frame: .zero)
+
+        if let identifierLabel = identifierLabel {
+            identifierLabel.adjustsFontForContentSizeCategory = true
+            identifierLabel.font = Fonts.subhead
+            identifierLabel.numberOfLines = 2
+            identifierLabel.text = identifier
+            identifierLabel.textAlignment = .natural
+            identifierLabel.textColor = Colors.Contact.firstName
```

Comment:
this looks like same as FirstName. Let's share the code, and perhaps just use one label (titleLable) for both